docs/articles/hosting/raspberry_pi.md

```diff
@@ -160,7 +160,7 @@ Since Mono neither is .NET Core, nor implements its APIs, this method will only
 Mono runtime has several caveats. It's notorious for being buggy, so this might not always work.
 
 Using your package manager, install Mono runtime (Debian/Raspbian: `sudo apt-get install mono-complete`). Once that is done, you will need to 
-follow the [Mono instructions](/articles/getting_started/mono.html "Mono instructions and notes") to complete the project setup.
+follow the [Mono instructions](/articles/beyond_basics/mono.html "Mono instructions and notes") to complete the project setup.
```

Comment:
The whole Raspberry Pi guide is very outdated, particularly since Pi1 and Pi0 models are very rare nowadays, and ARMv7 and I think even AArch64 are supported when publishing. Since we do not support mono, I would simply replace anything that requires mono (such as the Pi1/Pi0 guide) with a note that mono is no longer supported.

Reply:
Tonight Tyler and I were gonna do a run to remove anything win 7 and below .net 4.6.1 so this will be removed then